config/initializers/dynamoid.rb

```diff
@@ -8,7 +8,7 @@
   config.access_key = ENV["AWS_ACCESS_KEY_ID"]
   config.secret_key = ENV["AWS_SECRET_ACCESS_KEY"]
   config.region = "eu-west-2"
-  config.logger.level = :error
+  config.logger.level = :warn
```

Comment:
I think the problem here is that `config.logger` is a reference to the global rails logger.

So this line is setting the log level _everywhere_. At the moment, validation errors are logged at info, so you still won't get them with this.

Reply:
:thinking: humm, ok back to the drawing board on that one then.
Currently it's set to error same as the vulnerable people form.

I think i need to be a bit clearer on the state we want (sounds like, no PII but getting validation errors in logs), and where we should/souldn't see those.

Can catch up when we've got the more important stuff out of the way?

Reply:
Sure thing!